std/traits.d

```diff
@@ -164,7 +164,7 @@
  */
 module std.traits;
 
-import std.typetuple; // TypeTuple
```

Comment:
`staticMap` was originally in `std.traits`, until 5c9725d6cdfd3ffbeb4a2b8b8745ed920dee52c2 moved it to another module. So, breaking this would require going through a deprecation cycle.

Sidenote: ae was added to the project tester just a few days ago, I'm happy it caught a potential regression already :)